openwisp_monitoring/device/templates/admin/config/device/change_form.html

```diff
@@ -153,6 +153,22 @@ <h2>{% trans 'Interface status' %}: {{ interface.name }}</h2>
                           {{ interface.wireless.frequency }} GHz
                       </div>
                   </div>
+                  {% if interface.wireless.htmode %}
+                  <div class="form-row">
+                    <label>{% trans 'WiFi Version' %}:</label>
+                    <div class="readonly">
+                    {% if 'NOHT' in interface.wireless.htmode %}
+                        Legacy mode ({{ interface.wireless.htmode }})
+                    {% elif 'HE' in interface.wireless.htmode %}
+                        802.11ax/wifi6 ({{ interface.wireless.htmode }})
+                    {% elif 'VHT' in interface.wireless.htmode %}
+                        802.11ac/wifi5 ({{ interface.wireless.htmode }})
+                    {% elif 'HT' in interface.wireless.htmode %}
+                        802.11n/wifi4 ({{ interface.wireless.htmode }})
+                    {% endif %}
+                    </div>
+                  </div>
+                  {% endif %}
```

Comment:
I think all the logic above should be moved to the `data_user_friendly` method: https://github.com/openwisp/openwisp-monitoring/blob/master/openwisp_monitoring/device/base/models.py#L61-L100 .
This will allow us to keep the logic in a method and maintain this template simple, eg:

```python
if 'wireless' in interface and 'htmode' in interface['wireless']:
    interface['wireless']['htmode'] = self._get_wifi_version(interface['wireless']['htmode'])


def _get_wifi_version(self, htmode):
    # python logic here ....

Reply:
sure